src/containers/sidebar/SidebarContainer.ts

```diff
@@ -22,10 +22,15 @@ export const SidebarContainer = createContainer<SidebarValue>(() => {
   const worker = useContext(WorkerContext);
   const entities = EntityContainer.useContainer();
 
-  const [newItems, setNewItems] = useState(toStructualItems(entities.newItems));
-  const [passedItems, setPassedItems] = useState(toStructualItems(entities.passedItems));
-  const [failedItems, setFailedItems] = useState(toStructualItems(entities.failedItems));
-  const [deletedItems, setDeletedItems] = useState(toStructualItems(entities.deletedItems));
+  const defaultNewItems = toStructualItems(entities.newItems);
+  const defaultPassedItems = toStructualItems(entities.passedItems);
+  const defaultFailedItems = toStructualItems(entities.failedItems);
+  const defaultDeletedItems = toStructualItems(entities.deletedItems);
```

Comment:
Since the SidebarContainer function is called multiple times, I want to improve performance by memorize (`useMemo()`) :pray:

```suggestion
  const defaultDeletedItems = useMemo(() => toStructualItems(entities.deletedItems), [entities.deletedItems]);
```

Reply:
Looks good :) thanks